composer/platformatic.json

```diff
@@ -1,5 +1,15 @@
 {
   "$schema": "https://schemas.platformatic.dev/@platformatic/composer/2.62.1.json",
+  "server": {
+    "http2": true,
+    "https": {
+      "allowHTTP1": true,
+      "requestCert": false,
+      "rejectUnauthorized": false,
+      "key": { "path": "server.key" },
+      "cert": { "path": "server.crt" }
+    }
+  },
```

Comment:
This should be added to the server block of the main `watt.json` file

Reply:
I can't. The schema (`https://schemas.platformatic.dev/wattpm/2.62.1.json`) doesn't allow passing `https` values apart from `key` and `cert`

Reply:
Done on [d3e6948](https://github.com/rozzilla/watt-poc/pull/3/commits/d3e694816cebdb8eb008bd88051d67de60983e77), but I still get the same error